api/src/main/resources/Reference_Application_Diagnoses-11.xml

```diff
@@ -1,240 +1,240 @@
 <?xml version='1.0' encoding='UTF-8'?>
```

Comment:
Where do we use this file?

Reply:
@dkayiwa I am not sure tbh. It is mentioned as a step in the [release process wiki](https://wiki.openmrs.org/display/docs/OpenMRS+Reference+Application+Release+Process) .
The step states the following:
"Go to Administration -> Metadata Sharing -> Export Metadata and choose a package (repeat b. and c. for the following packages: Reference Application Concepts, Reference Application Diagnoses, Reference Application Order Entry and Allergies Concepts)"

Reply:
That is why I made these changes